controllers/lists_controller.go

```diff
@@ -26,7 +26,7 @@ type EditListForm struct {
 	Action string `param:"action"`
 }
 
-type ListJsonFormat struct {
+type ListJSONFormat struct {
```

Comment:
__[golint]__
* [golint] exported type ListJSONFormat should have comment or be unexported